app/mailers/project_summary_mailer.rb

```diff
@@ -0,0 +1,7 @@
+class ProjectSummaryMailer < ApplicationMailer
+
+  def project_summary(user)
+    @user = user
+    mail(to: user.email, subject: "[Learn IT, Girl] Daily summary of your board", sent: DateTime.now)
```

Comment:
Umm I dont know how to add date in subject :P SO i used this. It display the date on which the mail is sent.